joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/frontendspecific/rubysrc2cpg/ImplicitRequirePass.scala

```diff
@@ -7,23 +7,36 @@ import io.shiftleft.codepropertygraph.generated.{Cpg, DispatchTypes, EdgeTypes,
 import io.shiftleft.passes.ForkJoinParallelCpgPass
 import io.shiftleft.semanticcpg.language.*
 import io.shiftleft.semanticcpg.language.operatorextension.OpNodes
+import io.shiftleft.semanticcpg.language.operatorextension.OpNodes.FieldAccess
 import org.apache.commons.text.CaseUtils
 
 import java.util.regex.Pattern
 import scala.collection.mutable
 
+/** A 3-tuple holding the (name, fullName, importPath) for types in the analysis.
+  */
+type TypeImportInfo = (String, String, String)
```

Comment:
Please use a case class for this otherwise the usages sides of these triples will be cluttered with meaningless _1, _2...